src/switchroot/ostree-prepare-root.c

```diff
@@ -174,17 +174,19 @@ pivot_root(const char * new_root, const char * put_old)
 int
 main(int argc, char *argv[])
 {
-  const char *root_mountpoint = NULL;
+  const char *root_mountpoint = NULL, *root_arg = NULL;
   char *deploy_path = NULL;
   char srcpath[PATH_MAX];
   struct stat stbuf;
 
   if (argc < 2)
-    root_mountpoint = "/";
+    root_arg = "/";
   else
-    root_mountpoint = argv[1];
+    root_arg = argv[1];
 
-  root_mountpoint = realpath (root_mountpoint, NULL);
+  root_mountpoint = realpath (root_arg, NULL);
```

Comment:
I added it originally because I was defaulting argv[1] to `/sysroot` and I had a symlink `sysroot -> .` in my rootfs.  `realpath` was to resolve this symlink.  I guess it's not necessary with the way `ostree-prepare-root` is invoked now.